apps/apps.c

```diff
@@ -423,7 +426,16 @@ static char *app_get_pass(const char *arg, int keepbio)
                 return NULL;
             }
         } else {
-            BIO_printf(bio_err, "Invalid password argument \"%s\"\n", arg);
+            /* argument syntax error; do not reveal too much about arg */
+            tmp = strchr(arg, ':');
+            if (tmp == NULL || tmp - arg > PASS_SOURCE_SIZE_MAX)
+                BIO_printf(bio_err,
+           "Invalid password argument, missing ':' within the first %d chars\n",
```

Comment:
The indentation looks wrong to me here. Probably the string should be lined up with `bio_err`. If that won't fit within the 80 char line limit then you could break the string over two lines, e.g.
````
BIO_printf(bio_err,
           "Invalid password argment, missing ': '"
           "within the first %d chars\n",
           PASS_SOURCE_SIZE_MAX + 1);
````

Reply:
Well, this is what one would usually do, but I tried strictly following https://www.openssl.org/policies/codingstyle.html which says: 

> The limit on the length of lines is 80 columns. [...]
> Never break user-visible strings, however, because that breaks the ability to grep for them.

I also see the value in not breaking such strings.

Therefore, my resolution was to compromise indentation. Or what else to do to resolve the conflicting requirements? Go beyond the 80 chars width in such cases?
I suggest clarifying the coding guidelines in this respect.

Reply:
Interesting! I didn't know about that statement. It seems that either we have to break the coding style's indentation rules, or the coding styles rules on not breaking user visible strings, or the coding styles rules on line length. One way or another the rules have to be broken in this case! I think the least offensive option (to me at least) is to indent it properly and break the 80 char limit.

Reply:
I tend to agree that sacrificing the line length limit hurts least. Done.

Reply:
Hmmm...  thing is, we have already broken strings into pieces elsewhere.  I was also unaware of the no-break rule (how did that happen?), was that actually our own idea, or does that come from the Linux guidelines (that our coding style originates from)?

Reply:
@mattcaswell, thanking also into account #6254, can we consider your request to change the indentation fulfilled by my last commit?

Reply:
ping @mattcaswell

Reply:
Sorry, I missed this comment - thanks for the ping. This seems ok now, but there is a similar styling nit a couple of lines later on - indentation off by 1 character in the second error message. Other than that this PR looks good to go.

Reply:
Oh, I recently found finalization of this PR has been held up for many months because I forgot to fix this very minor issue. I've just rebased the PR and added the missing space character.